pkgs/by-name/cm/cmake/005-remove-systemconfiguration-dep.diff

```diff
@@ -15,15 +15,15 @@ diff -Naur cmake-3.28.1/Utilities/cmcurl/CMakeLists.txt cmake-3.28.1-new/Utiliti
    endif()
  endif()
  
-diff -Naur cmake-3.28.1/Utilities/cmcurl/lib/curl_setup.h cmake-3.28.1-new/Utilities/cmcurl/lib/curl_setup.h
---- cmake-3.28.1/Utilities/cmcurl/lib/curl_setup.h	2023-12-14 10:47:27.000000000 -0500
-+++ cmake-3.28.1-new/Utilities/cmcurl/lib/curl_setup.h	2024-01-08 14:45:08.000000000 -0500
-@@ -255,12 +255,6 @@
+diff -ru cmake-3.28.2/Utilities/cmcurl/lib/curl_setup.h cmake-3.28.2-patched/Utilities/cmcurl/lib/curl_setup.h
+--- cmake-3.28.2/Utilities/cmcurl/lib/curl_setup.h	2024-01-29 21:01:32
++++ cmake-3.28.2-patched/Utilities/cmcurl/lib/curl_setup.h	2024-01-31 21:20:50
+@@ -255,12 +255,7 @@
   * performing this task will result in a synthesized IPv6 address.
   */
  #if defined(__APPLE__) && !defined(USE_ARES)
 -#include <TargetConditionals.h>
--#define USE_RESOLVE_ON_IPS 1
+ #define USE_RESOLVE_ON_IPS 1
```

Comment:
Being closer to upstream again seems like an improvement, but since I didn't see any comment about this in the thread I wonder if this change was intentional?

Reply:
You're right, it was not intentional. Let me remove that line in the patch again.